lib/msf/core/post_mixin.rb

```diff
@@ -59,7 +59,10 @@ def check_for_session_readiness(tries=6)
       select(nil,nil,nil,back_off_period)
     end
     session_ready = !!session.sys
```

Comment:
So this basically requires that `stdapi` be loaded before any post modules can be run which I guess makes sense. Only module that use the core API which is just meta-commands for things like transports would function without the stdapi extension. While I found it surprising, I left it this way and just changed the error message because I found "Could not get a hold of the session." to be confusing when I knew I had an established session without stdpai loaded.

This checks if the TLV encryption key is nil to differentiate between stdapi not being loaded and the entire session not being fully established yet. This is the same technique I used for session verification in #14789.